### PR TITLE
Update: Inputs and Images

### PR DESCRIPTION
**Pull request by HeyNitin**
Added Inputs and Images.
Also improved css in other file

### CODE REVIEW

**Comment by tanishk15sharma** [comment]
nice , looking good to me

**Review by tanishk15sharma — APPROVED** [review]

File: Colors/colors.css [hunk 1]
```diff
@@ -1,4 +1,10 @@
 :root {
   --background-color: rgb(219, 53, 178);
   --primary-color: grey;
+  --success: green;
+  --warning: rgb(209, 148, 35);
+  --error: red;
```
Comment (Naveenchandar) [inline Colors/colors.css:6]: Please use hex code for consistency

File: Components/images/images.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+</head>
+
+<body>
+
+    <div>
+        <img class="img-responsive" src="../../Assets/imgs/male profile.jpg" alt="">
+    </div>
+
+    <div>
+        <img class="img-round" src="../../Assets/imgs/male profile.jpg" alt="">
```
Comment (Naveenchandar) [inline Components/images/images.html:19]: Please provide alt description for accessibility

File: Components/images/images.css [hunk 3]
```diff
@@ -0,0 +1,15 @@
+.img-round {
+
+    height: 10rem;
+    width: 10rem;
+    border-radius: 10rem;
+    object-fit: cover;
+
+}
```
Comment (rohanmathur91) [inline Components/images/images.css:8]: remove the blank lines

**Review by rohanmathur91 — APPROVED** [review]

File: Components/images/images.css (hunk 3 above)
Comment (rohanmathur91) [reply Components/images/images.css:8]: Rest looks good

**Pull request merged**